packages/calcite-components/src/components/alert/alert.scss

```diff
@@ -4,13 +4,17 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-alert-width: Specifies the width of the component.
+ * @prop --calcite-alert-background-color: Specifies the component's background color.
```

Comment:
Where are these tokens applied?

Reply:
`--calcite-alert-width` is being applied in `.container`:
`inline-size: var(--calcite-alert-width, auto);` 

and in `scales`:
`inline-size: var(--calcite-alert-width, 40rem);`

`--calcite-alert-background-color` is being applied in `.container`:
`background-color: var(--calcite-alert-background-color, var(--calcite-color-foreground-1));`